util/include/Digest.hpp

```diff
@@ -14,17 +14,21 @@
 #include <memory.h>
 #include <stdint.h>
 #include <string>
-#include "DigestType.h"
+#include <array>
+#include "DigestType.hpp"
 
-namespace bftEngine {
-namespace impl {
+namespace concord::util::digest {
+
+using BlockDigest = std::array<std::uint8_t, DIGEST_SIZE>;
```

Comment:
BlockDigest definition should be part of kvbc, utils knows nothing about blocks

Reply:
Moving this definition to `kvbc/include/categorization/blocks.h`